### ISSUE TITLE
VLAN Virtual NIC support

### ISSUE THREAD

**Issue by bghite** [KurtisLiggett/Simple-IP-Config#20]
Hello,

Utility is not able to properly access Intel Virtual NICs that are used to support VLANs.  Due to not being able to access properties, address cannot be set on Virtual VLAN NICs.
 
Behaviour observed: No settings appear in "Current Adapter Properties"
Environment: Windows 10 Version 1703
NIC Drivers: Intel Wired Proset Version 22.4.0.1

**Comment by KurtisLiggett** [comment]
Does the NIC get detected and added to the dropdown selection?

This is probably a limitation or missed option in the [Windows IP Helper API](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365798(v=vs.85).aspx).

I seem to remember picking which types of adapters I requested info for, so maybe it is just a parameter I missed. I'll take a look when I have time.

**Comment by bghite** [comment]
Yes. The Virtual VLAN NICs do get properly detected and added to the dropdown selection.

**Comment by KurtisLiggett** [comment]
I made a test program to help troubleshoot this issue. If you could, try running the program linked below and post a screenshot of the message. It should be similar to this.

![screenshot from 2017-08-30 14-47-33](https://user-images.githubusercontent.com/29669372/29935117-4ab632e4-8e43-11e7-8202-99dbf7194e6b.png)



[getAdapters_Test.zip](https://github.com/KurtisLiggett/Simple-IP-Config/files/1267888/getAdapters_Test.zip)
Source files are included in the zip.

**Comment by bghite** [comment]
Here is the screenshot that you requested. Things seem to work correctly.
![screen capture](https://user-images.githubusercontent.com/29780223/29938362-5c4252dc-8e3d-11e7-958e-0128c55e97b4.PNG)

**Comment by KurtisLiggett** [comment]
It's strange that your Ethernet2 and Ethernet3 from the top are not also found in the bottom list.
Can you post a screenshot of your network connections window?

Is it possible for me to recreate these "Virtual VLAN NICs" on my PC for troubleshooting?

**Comment by bghite** [comment]
You are right that is interesting. You can create the Virtual VLAN NICs if you have an Intel network card and the latest Intel Drivers. I am using ver22.4.0.1 of the Intel Wired Proset drivers.

Here is a screenshot of my network connections:
![network connections](https://user-images.githubusercontent.com/29780223/29942121-99c30e06-8e49-11e7-9209-67bdcdcc32db.PNG)

**Comment by elmorekevin** [comment]
I just discovered the same issue, though I have a Realtek NIC. I'd like to add to this post if I may. To add a VLAN, I have to run the Realtek Ethernet Diagnostic Utility. This then adds another network adapter to the system (I called mine VLAN 100). Unfortunately realtek hasn't designed their software to add a VLAN through the Controller Properties>Advanced menu.

Once I add a VLAN adapter, it does detect in SIC, and I was able to set an IP on it successfully. Once I did this though, I wasn't able to see the current adapter properties of the initial/standard ethernet adapter.

Here's a screen shot using the program included above:
![image](https://user-images.githubusercontent.com/31101710/30326024-19899f52-9795-11e7-8fec-aa7a9a5e44db.png)
